software/nginx/en.md

```diff
@@ -0,0 +1,30 @@
++++
+title = "nginx"
+lastmod = "2020-04-03T23:40:51+02:00"
++++
+# nginx
+Solus provides the nginx web server and proxy to users and developers who want to test their web applications before setting them to production. \
+Remember that Solus is not a server Operating System: any usage outside the local testing is not encouraged.
```

Comment:
operating system* and instead if "is not encouraged" it should be "is discouraged".